src/Collector/HttpClientInterfaceProxy.php

```diff
@@ -28,7 +28,8 @@ public function sendRequest(RequestInterface $request): ResponseInterface
         } finally {
             $endTime = microtime(true);
             $this->collector->collectTotalTime($response, $endTime, $uniqueId);
-            return $response;
         }
+
+        return $response;
```

Comment:
Is it OK if this line won't be reached in case of an exception?

Reply:
Return into `finally` block suppress all exceptions. It's bug in this case.